projects/logging/examples/logging.yml

```diff
@@ -2,7 +2,7 @@ kernel:
   image: "linuxkit/kernel:4.9.x"
   cmdline: "console=ttyS0 console=tty0 page_poison=1"
 init:
-  - linuxkit/init:1b8a7e394d2ec2f1fdb4d67645829d1b5bdca037 # with runc, logwrite, startmemlogd
+  - linuxkit/init:b5c88b78cd9cc73ed83b45f66bc9de618223768a # NOTE: custom init with logwrite ctr/runc, startmemlogd
```

Comment:
Thanks @riyazdf and @justincormack. I think I'll close this PR for now and make a new one that points to the new repos, as I also have to update memlogd to use the latest version of the init scripts